react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+
+
+import { connect } from 'react-redux';
+import { arrayOf } from 'prop-types';
+
+import actionsCreators from '../../../../../redux/table-matches/actions';
+
+import styles from './styles.module.scss';
+
+var Spinner = require('react-spinkit');
+
+class TableMatches extends Component {
+  componentDidMount() {
+    this.props.getMatches();
+  }
+
+  renderMatch(match) {
+    return (
+      <tr>
+        <td className={styles.tableChild}>{match.player_one}</td>
+        <td className={styles.tableChild}>{match.player_two}</td>
+        <td className={styles.tableChild}>{match.winner === 'player_one' ? match.player_one : match.winner === 'player_two' ? match.player_two : match.winner}</td> 
+      </tr>
+    );
+  }
+
+  renderLoading() {
+    return <Spinner className={styles.loading} name='double-bounce' />;
+  }
+
+  render () {
+    if (this.props.matches.length) {
```

Comment:
this if/else could be done using the ternary operator. Also we usually destructure `this.props` to avoid repeating.
```jsx
const { matches } = this.props;
return matches.legth ? (<div> ... </div>) : (<div> ... </div>);
```